dependencies.yml

```diff
@@ -11,7 +11,9 @@ boms:
   - io.netty:netty-bom:4.1.50.Final
   - io.zipkin.brave:brave-bom:5.11.2
   - org.eclipse.jetty:jetty-bom:9.4.28.v20200408
+  - org.springframework.boot:spring-boot-dependencies:2.3.0.RELEASE
```

Comment:
Since Spring Boot 2.3.0, the dependency libraries info included in spring-boot has been changed.
spring-boot doesn't have dependency libraries and version.
I guess we need to refer to spring-boot-dependencies.

2.2.7:
- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot/2.2.7.RELEASE
- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/2.2.7.RELEASE
- https://github.com/spring-projects/spring-boot/blob/v2.2.7.RELEASE/spring-boot-project/spring-boot-dependencies/pom.xml

2.3.0:
- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot/2.3.0.RELEASE
- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/2.3.0.RELEASE
- https://github.com/spring-projects/spring-boot/blob/v2.3.0.RELEASE/spring-boot-project/spring-boot-dependencies/build.gradle

Reply:
I think adding `spring-boot-dependencies` is not a good idea. There are so many version conflicts with other modules such as dropwizard. Actually that's the reason we didn't use Spring BOM before.

Reply:
So the solution is I think we need to add the needed dependencies one by one.